desktop/server.js

```diff
@@ -21,10 +21,12 @@ if (NO_SERVER) {
     if (jsonStats.errors.length > 0) {
       throw jsonStats.errors.join('\n')
     }
+
+    console.log(stats)
+
     if (jsonStats.warnings.length > 0) {
```

Comment:
so this only shows up if you do a ```npm run build-dev``` so while devving we likely won't see it so CI is likely a good answer there. That seems fine to me if you guys are up for it. it means if that happens either

1. Real files got orphaned and should be deleted
1. or we added a file webpack doesn't know about (like flow-only files) and we need to add to the ignore list

if that seems ok to you guys lets ticket it